v2/api.go

```diff
@@ -95,15 +94,14 @@ func (api *APIConfiguration) ExecuteRequest(req *http.Request) ([]byte, error) {
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("User-Agent", "courier-go/"+api.SDKVersion)
```

Comment:
We may want to set this to something like `"merit/courier-go/" + api.SDKVersion`, since we're using a modified version of the client.

Reply:
Their documentation doesn't mention the header, and I wonder what they're using it for. I'm going to leave it for now just in case they use it for something and they start denying requests over the header not being what they expect.